README.md

```diff
@@ -69,6 +69,22 @@ Install Kerberos client to enable `skt` authentication to Beaker:
 Install `skt` dependencies following its
 [README.md](https://github.com/RH-FMK/skt).
 
+Run tests
```

Comment:
Going with @spbnick 's comment about standalone commits, the section about test runs should be added in the commit that deals with tests.

Reply:
Do I need to squash the commit `add .pytest_cache to gitignore` and `Add run tests section` into one commit?

Reply:
No, those two changes are unrelated to each other.

Reply:
Okay. could you review now?